dev/com.ibm.ws.org.glassfish.json.2.0/bnd.overrides

```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+
+Bundle-SymbolicName: com.ibm.ws.org.glassfish.json.2.0
+
+Export-Package: \
+   org.glassfish.json.*;version="1.0.17"
```

Comment:
This package export should be at version `2.0.0` instead of `1.0.17`

Reply:
Sure.  Currently this bundle project is vestigial, fyi.  A future change will wire this bundle into jsonp-2.0 feature support.